### DISCUSSION TITLE
Paulmann none of the URail Components yet supported?

### DISCUSSION THREAD

**Discussion by SVNKoch** [Koenkk/zigbee2mqtt#9832] [Devices]
I've not yet bought the device, but maybe will next month.
So Paulmann has the rail system for it's LED lights. There are some componetns which support Zigbee, including:
Zigbee Switch: [50050 / 50051 / 96815](https://en.paulmann.com/p/urail-rail-adapter-smart-home-zigbee-dimm-switch-155x56mm-white/50051)
Power Supply: [50120 / 50121 / 96819](https://en.paulmann.com/p/urail-power-supply-mitte-227x196mm-max.-300w-white/50121)
Rail Adapter: [96893](https://en.paulmann.com/p/urail-rail-adapter-smart-home-zigbee-on-off-dimm-166x20mm-chrome-matt/96893)
Spot:  [95525 / 95526](https://en.paulmann.com/p/urail-rail-spot-smart-home-zigbee-cone-individual-spot-incl.-tunable-white-illuminant-gu10-330lm-5w-2700-6500k-230v-chrome-matt-chrome/95526)

None of which seem to be yet supported. But couldn't find any more infos on them regarding z2m.
Any got any information on them? Should be safe to buy and add to DB then, right (especially 50051 :wink: )?

**Comment by gritche2** [comment]
Hello @SVNKoch 

do you have bu ythis system ?
I have it : it was working on my deconz and I have issues during my migration.

Regards

**Reply by SVNKoch** [reply]
I couldn't yet test it unfortunately. Buying them however is still on the list if things to do at some point. But I can't imagine them making a big problem getting paired and added as new device tbh

**Reply by gritche2** [reply]
At the end I success to connect it. Guide for unsupported device is good and some of paulmann has been already added.
Thanks
